setup.py

```diff
@@ -58,6 +58,7 @@
     'invenio-logging>=1.0.0a3',
     'invenio-mail>=1.0.0a4',
     'invenio-oauthclient>=1.0.0a8',
+    'invenio-query-parser>=0.6.1.dev20160418',
```

Comment:
Checking the [commits](https://github.com/inveniosoftware/invenio-query-parser/commits/master) in `invenio-query-parser` this might be `'invenio-query-parser>=0.6'`